.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+---
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os:
+          - ubuntu-24.04
+          - windows-2022
+          - macos-13
+          - macos-14
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+      - name: Read .promu.yml
+        uses: pietrobolcato/action-read-yaml@dd664040f4883322f6d143e58302062b35a46e4d
```

Comment:
if possible, we would like to avoid using third party actions, and would prefer to only use actions from `actions` and `prometheus` orgs.

Reply:
I would like do this the way Prometheus does for consistency across repos in Prometheus org, we already have a similar comment for circleci.

https://github.com/prometheus/blackbox_exporter/blob/7d61fee99ac4727c9669413c685356ea2da1d621/.promu.yml#L2